packages/core/src/command/consolidated/get-snapshot-local.ts

```diff
@@ -189,26 +193,23 @@ async function postSnapshotToApi({
   );
 }
 
-export function removeContainedPatients(bundle: Bundle, patientId: string): Bundle {
-  if (!bundle.entry) return bundle;
-
-  const updatedEntry = bundle.entry.map(entry => {
-    const resource = entry.resource;
-    if (resource && "contained" in resource) {
-      return {
-        ...entry,
-        resource: {
-          ...resource,
-          contained: resource.contained?.filter(r => !isPatient(r) || r.id === patientId),
-        },
-      };
-    }
-    return entry;
+export function removeContainedPatients(bundle: Bundle, patientIdToKeep: string): Bundle {
+  return removeContainedResources({
+    bundle,
+    shouldRemove: (r: Resource) => isPatient(r) && r.id !== patientIdToKeep,
   });
```

Comment:
Moved to `removeContainedResources` in `packages/core/src/external/fhir/bundle/remove.ts`